code/addons/interactions/package.json

```diff
@@ -30,6 +30,7 @@
     },
     "./manager": "./dist/manager.js",
     "./preview": "./dist/preview.js",
+    "./preset": "./dist/preset.js",
```

Comment:
Could you check if this is missing in more addons? If so, correct those in this PR as well? 🙏